src/scripts/main.js

```diff
@@ -1,3 +1,391 @@
 'use strict';
+'use strict';
```

Comment:
There are two 'use strict' directives. Only one is necessary at the top of the file.